data/data/bootstrap/bootstrap-in-place/files/opt/openshift/bootstrap-in-place/master-update.fcc

```diff
@@ -45,6 +45,10 @@ storage:
       contents:
         local: bin/installer-gather.sh
       mode: 0555
+    - path: /opt/openshift/original_cvo_overrides.patch
+      contents:
+        local: original_cvo_overrides.patch
+      mode: 0555
```

Comment:
nit: no need for the executable bit on this `*.patch` file.  Looks like the precedent here is for `*.sh` to get the executable bit (makes sense), but for non-executables to leave off `mode` entirely, so perhaps we can just drop this line?

Reply:
Right, just copy paste mistake